tests/test_fields.py

```diff
@@ -50,7 +50,7 @@ def test_get_value_from_instance_with_unknown_attr(self):
         class Dummy:
             attr1 = "foo"
 
-        field = DEDField(attr='attr2')
+        field = DEDField(attr='attr2', required=True)
```

Comment:
Any particular reason to add it to required ?

Reply:
The test tries to assert `VariableLookupError` is raised. but `VariableLookupError` is not raised if required is not True.
This test also fails locally.

https://github.com/django-es/django-elasticsearch-dsl/blob/759076bb8e968db311610226966d448767e4441e/django_elasticsearch_dsl/fields.py#L71-L75